tta-server/app/controllers/GameStateResource.scala

```diff
@@ -4,11 +4,7 @@ import javax.inject.Inject
 import javax.inject.Singleton
 
 import logic.Logic
-import models.ActionId
-import models.Bronze
-import models.GameState
-import models.PlayerIndex
-import models.PlayerState
+import models._
```

Comment:
don't use wildcard imports, for reasons described here https://docs.dkandu.me/practices/languages/scala/imports.html

IntelliJ IDEA > Prefrerences > Code Style > Scala > Imports lets you configure intellij to not automatically do this:
![screen shot 2015-01-22 at 9 04 35 am](https://cloud.githubusercontent.com/assets/3209204/6995688/ab6186e4-db0c-11e4-95b9-edf18923a1fd.png)